main/python-pycups/template.py

```diff
@@ -0,0 +1,20 @@
+pkgname = "python-pycups"
+pkgver = "2.0.1"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+    "python-wheel",
+]
+makedepends = ["cups-devel", "python-devel"]
+depends = ["cups", "python"]
```

Comment:
should be cups-libs here

Reply:
or maybe not even that, it should get autodetected via shlibs

Reply:
and it does:

```
=> python-pycups-2.0.1-r0: running pre_pkg hook: 001_runtime_deps...
   SONAME: libcups.so.2 <-> cups-libs
```

Reply:
I'm already aware that `cups-libs` is autodetected as an shlib dependency, but I added a `cups` dependency because [the README states](https://github.com/OpenPrinting/pycups/blob/3a661e382d6ab8404c9d9327bf71e85abe154f42/README):

> CUPS needs to be installed and running for PyCUPS working correctly.

Though I _think_ it'd work regardless? I can still remove it if you think that's the right thing to do.

Reply:
looking at other distros, they don't have it, and it does not make sense to have it considering this is API bindings to libcups and doing this will make installing gnome pull in an entire cups server (which should be user choice)

Reply:
e.g. https://packages.debian.org/sid/python3-cups